src/cbc_sdk/platform/alerts.py

```diff
@@ -979,6 +980,25 @@ def most_recent_alert_(self):
         """
         return self._most_recent_alert
 
+    def get_alerts(self):
```

Comment:
Since this returns the alert query object rather than a list of alerts, I think either
* the method name should be changed to get_alerts_query
or
* change the method should return a list.  `return alert_search_query.all()` will achieve this.

Agree it works as designed, but isn't intuitive to use.